manageiq-loggers.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "activesupport",     ">= 5.0"
+  spec.add_runtime_dependency "cloudwatchlogger",  "~> 0.2"
```

Comment:
Also, as discussed, let's drop the optional dependencies in favor of a README entry.